cocotb/decorators.py

```diff
@@ -40,6 +40,7 @@
 from cocotb.utils import get_sim_time, lazy_property
 from cocotb import outcomes
 from cocotb import _py_compat
+from cocotb import scheduler
```

Comment:
Perhaps:
```suggestion
# renamed to avoid confusion with the scheduler instance `cocotb.scheduler`
from cocotb import scheduler as scheduler_module
```

Reply:
```suggestion
```